client/src/App.css

```diff
@@ -36,3 +36,59 @@
     transform: rotate(360deg);
```

Comment:
Would like to see some comments for the CSS as well. Possibly breaking out into component or page sections at a minimum.